v3/src/components/slider/slider-model.ts

```diff
@@ -53,11 +53,17 @@ export const SliderModel = TileContentModel
       }
       self.globalValue.setValue(n)
     },
-    setMultipleOf(n: number) {
-      self.multipleOf = Math.abs(n)
+    setMultipleOf(n: number | string) {
+      const multipleOf = typeof n !== "number" ? parseFloat(n) : n
```

Comment:
This sort of input parsing should be the client's responsibility, not the model's.